### ISSUE TITLE
Eingabe von Gerät/Personal auch nach Aufruf des Änderungslinks

### ISSUE THREAD

**Issue by ifeelit** [COVID19-PIRAT/Konzept#40]
Bei der ersten Eingabe der Kontaktperson hab ich nicht gecheckt, dass man unten noch die Links anklicken muss, um Geräte oder Personal hinzuzufügen. Nach Eingabe kann man das aber nicht mehr machen, auch über den Änderungslink (Email) nicht. Wenn man also jetzt noch ein Gerät eingeben will muss man alles von vorne machen.

**Comment by chaoran-chen** [comment]
Wie ist die Priorität? Wie wichtig ist es, dass es bis zum Release fertig ist?

Der aktuelle Plan ist nämlich fürs Erste die Angebotbearbeitungsseite in eine Angebotübersichtsseite umzuwandeln mit dem Hinweis, dass Löschungs- und Änderungswünsche über eine Emailanfrage geäußert werden sollen.

Für das Projekt ist es wesentlich, dass es einfach ist, Angebote reinzustellen. Dass das Löschen bisschen komplizierter ist, fände ich nicht so dramatisch.

**Comment by ifeelit** [comment]
Das issue an sich muss nicht bis zum Release fertig sein, ich würde dann aber zumindest einen Hinweis geben, vor dem Abschicken, dass man noch kein Personal, Gerät oder Verbrauchsmittel eingegeben hat. Was meint ihr?

**Comment by ifeelit** [comment]
Vielleicht kann man die Ressourcen als Checkboxes realisieren, dann kann man sie auch wieder abwählen, falls man eine falsche Ressource angeklickt hat?

**Comment by chaoran-chen** [comment]
Stimmt, eine Anfrage ganz ohne Ressourcen ergibt keinen Sinn. Wir können einführen, dass man mindestens eine Ressource eintragen muss.

Man kann ja die falschausgewählte Ressource löschen, indem man auf den Mülleimer klickt. Oder meinst du ein anderes Problem?

![image](https://user-images.githubusercontent.com/18666552/77547129-17728480-6ead-11ea-8287-1e5ed77f8839.png)

**Comment by ifeelit** [comment]
Ah ok, dann passt das mit dem Löschen.
